meson.build

```
@@ -686,68 +676,58 @@ else
     error('Mandatory header zlib.h not found!')
   endif
   icu_dep = dependency('icu-uc')
-  syslua = get_option('syslua')
-  if syslua != 'false'
-    lua_dep_tmp = dependency('lua-5.4', 'lua-54', 'lua54', 'lua5.4', required:false)
-  endif
 endif
 
 # Set unconditionally, as it was checked as hard requirement
 pub_conf_data.set('FREECIV_HAVE_LIBZ', 1)
 
-mw_req = get_option('mwand')
+mw_dep = dependency('MagickWand', 'MagickWand-6.Q16HDRI', required: get_option('mwand'))
```

... but why aren't you pruning this branch as soon as possible?

```suggestion
mw_dep = dependency('MagickWand', 'MagickWand-6.Q16HDRI', version: '>=6.0', required: get_option('mwand'))
```

Obviously you can additionally check if it narrows further to >=7.0 when determining which cfg data to set.